src/jdk.incubator.jextract/share/classes/jdk/internal/jextract/impl/JavaSourceBuilder.java

```diff
@@ -256,6 +263,147 @@ protected void emitGetter(String mods, Class<?> type, String name, String access
         emitGetter(mods, type, name, access, false, null);
     }
 
+    protected void emitFunctionWrapper(String mods, MethodType mtype, String javaName, String access,
```

Comment:
This was just moved "as is" from HeaderFileBuilder, so that ConstantBuilder could access it